src/saml.ts

```diff
@@ -278,14 +280,24 @@ class SAML {
       };
     }
 
+    const nameIDPolicy: XMLInput = {
+      "@xmlns:samlp": "urn:oasis:names:tc:SAML:2.0:protocol",
+    };
+
+    if (this.options.allowCreate != null) {
```

Comment:
This pull request introduces `options.allowCreate` as (see addition at line 157):
```javascript
      allowCreate: ctorOptions.allowCreate ?? true,
```
and related addition at `types.ts` is at the moment:
```javascript
allowCreate?: boolean;
```
IMHO usage of nullish coalescing operator means that value of `this.options.allowCreate` shall always be `false` or `true` but never `null` at this point of code execution flow. I.e. this `if` statement would always evaluate to to `true` (and thus it would be "NOP" check).